packages/main/src/TabContainer.js

```diff
@@ -500,16 +500,37 @@ class TabContainer extends UI5Element {
 		}
 
 		if (isEnter(event)) {
```

Comment:
The Enter and Space keys should do the same. Is it possible to remove the duplicated code and unify those checks? Upon pressing space, _onHeaderItemSelect doesn't get called, but when the enter key is pressed, it does. The code reads as if those two buttons would act differently.